backend/views/views.py

```diff
@@ -140,8 +140,8 @@ def delete_user(users_id):
     user = db.session.query(User).get(users_id)
     is_deleted = user.delete()
     db.session.commit()
-    msg = "User deleted" if is_deleted else "Impossible to delete user"
-    flash(msg)
+    if not is_deleted:
+        flash("Impossible to delete user, because it is the last admin!")
```

Comment:
maybe would be better something like "The last admin can't be deleted!" or "Impossible to delete the last admin"

Reply:
I agree so much better